nilearn/plotting/img_plotting.py

```diff
@@ -2171,7 +2172,7 @@ def plot_img_comparison(
             ax2.legend(loc="best")
 
             if output_dir is not None:
-                if not os.path.exists(output_dir):
+                if not Path.exists(output_dir):
                     os.makedirs(output_dir)
                 plt.savefig(os.path.join(output_dir, f"{int(i):04}.png"))
```

Comment:
note to self (not for this PR): the different error fixes may need to be further refactored.

Probably will have to be flagged by a grep for `Path.`

```suggestion
                output_dir = Path(output_dir)
                output_dir.mkdir(exists_ok=True)
                plt.savefig(output_dir / f"{int(i):04}.png")
```